pgsu/__init__.py

```diff
@@ -250,16 +250,18 @@ def _execute_psyco(command, dsn):
     """
     import psycopg2  # pylint: disable=import-outside-toplevel
 
+    conn = None
     output = None
-    with psycopg2.connect(**dsn) as conn:
+    try:
+        conn = psycopg2.connect(**dsn)
```

Comment:
the actual fix is here

Reply:
Thanks! 

could you please add a comment to that effect? 

i.e. "don't use a context manager so that psycopg2 does not start a transaction" ?

There is no variable one can pass to the context manager to tell it not to do this, right?

Reply:
No, not according to the issue I linked to from psycopg. As it says there, the contextmanager was not actually doing anything before (in v2.8) anyway